apps/sharedtest/test/unit/crop_resize_rotate_test.js

```diff
@@ -18,6 +100,10 @@ function runCropResizeRotateTests(imageWidth, imageHeight) {
     const W = imageWidth, H = imageHeight;  // The size of the test image
 
     suiteSetup(function(done) {
+
+      // We need the mock lazy loader for jpeg-exif.
+      window.LazyLoader = window.MockLazyLoader;
```

Comment:
I plan on making the metadata copy an explicit flag... so that by default it ain't. Then I'll need to hook it up in a couple of places then.